target/scripts/helpers/index.sh

```diff
@@ -10,6 +10,7 @@ function _import_scripts
   source "${PATH_TO_SCRIPTS}/accounts.sh"
   source "${PATH_TO_SCRIPTS}/aliases.sh"
   source "${PATH_TO_SCRIPTS}/change-detection.sh"
+  source "${PATH_TO_SCRIPTS}/dhparams.sh"
```

Comment:
This files contents could probably live in the `ssl.sh` helper as it's related to that support (ciphers)

Reply:
I will merge this PR (to get it through), and will apply this feedback in the next, smaller PR :)

Reply:
See #3124